.github/workflows/netflify.yml

```diff
@@ -0,0 +1,22 @@
+name: Netlify
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  trigger_builds:
+    name: Trigger netlify build preview
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
```

Comment:
Why do we checkout? We only want to trigger the builds right? Netlify will handle the rest?

Reply:
Because that was in the action I based it on, removed :+1: